cmd/krel/cmd/release_notes.go

```diff
@@ -167,6 +168,14 @@ func init() {
 		"add dependency report",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringSliceVarP(
+		&releaseNotesOpts.mapProviders,
+		"maps-from",
+		"m",
+		[]string{},
+		"release notes map providers",
+	)
```

Comment:
Okay cool, this either needs some documentation or we think about a more simple solution for now. For example, what if we just accept a list of `string` directories to search recursively for `*.y[a]ml` files?

Reply:
I could have built it simpler but we are already considering two new sources of data: manual fixes and the security data. The first maps we create will most likely be read from a directory while the security maps will come from a different source. That's why I was considering this notion of providers.

Now, the actual reading of notes does not necessarily have to be in the release notes package but I think it's better not to have to copy the maps first.

What I could do is have the `--maps-from` flag be a directory ___unless___ you need to read from another source. Like this:

```
--maps-from=/path/to/yamls  
```
and we could add to it the capability to understand urls or url-like strings for other sources:
```
--maps-from=gs://gcp-bucket/
--maps-from=github://org/secret-repo
```
wdyt ?

Reply:
Yes, sounds good. This way we can start simple and enhance complexity later on. :+1:

Reply:
The flag and the implementation are done, let me know about the `--help` hint. Maybe we could add more usage info in the large help text